torch/jit/__init__.py

```diff
@@ -953,6 +970,9 @@ def _try_compile_fn(fn):
         # Don't do anything for @ignore'd functions
         return None
 
+    if isinstance(fn, torch.nn.Module):
```

Comment:
Could you add a comment here? It's i don't understand why we're returning None for both ignores and nn.Modules